libs/plutus-preprocessor/plutus-preprocessor.cabal

```
@@ -30,9 +30,9 @@ executable plutus-preprocessor
         cardano-ledger-alonzo >=1.1,
         flat,
         hashable >=1.4.1.0,
-        plutus-tx ^>=1.1,
-        plutus-tx-plugin ^>=1.1.1,
-        plutus-ledger-api ^>=1.1,
+        plutus-tx ^>=1.5,
+        plutus-tx-plugin ^>=1.5,
+        plutus-ledger-api ^>=1.5,
```

We can remove this, since this package is never released to CHaP. But feel free to leave it as is if you like.
```suggestion
        plutus-tx,
        plutus-tx-plugin,
        plutus-ledger-api,
```